coverage/html.py

```diff
@@ -412,9 +421,14 @@ def write_html_file(self, ftr: FileToReport, prev_html: str, next_html: str) ->
                 )
             ldata.css_class = ' '.join(css_classes) or "pln"
 
+        di = file_data.__dict__
+        if contexts_json == "{}":
```

Comment:
It feels a little odd to check for no contexts like this.  Can we use `if not contexts:` instead?